### ISSUE TITLE
ZSH, iTerm: "command not found: qfc"

### ISSUE THREAD

**Issue by dewey** [pindexis/qfc#18]
Hello,

I just installed qfc and added it to my zsh 5.0.5 `.zshrc` like that:

```
[[ -s "$HOME/.qfc/bin/qfc.sh" ]] && source "$HOME/.qfc/bin/qfc.sh"
qfc_quick_command 'cd' '\C-b' 'cd $0'
qfc_quick_command 'vim' '\C-p' 'vim $0'
```

and restarted my terminal / sourced the config file.

If I press ctrl + f the result looks like this:

```
~|⇒
qfc_complete:15: command not found: qfc
```

Is there anything I missed?

Thanks

**Comment by lukechilds** [comment]
I've tried Term 3 and the default OS X terminal app with both zsh 5.1 and bash 3.1. I always get:

```
qfc_complete:20: command not found: qfc
```

☹️

**Comment by desyncr** [comment]
@dewey @lukechilds You need to add `$HOME/.qfc/bin/` to your path.

**Comment by lukechilds** [comment]
@desyncr I'm not sure what's changed since I last tried but I just tried again and it works fine now after sourcing, I don't need to add anything to my PATH.

`source "$HOME/.qfc/bin/qfc.sh"` is all I need.

**Comment by desyncr** [comment]
@lukechilds You are right. It's probably [this](https://github.com/pindexis/qfc/commit/0cb201c0f555d9b06949e566f80ccd803458f70a).

**Comment by lukechilds** [comment]
@desyncr Good spot 👍 

@dewey @pindexis This can probably be closed.
